app/src/main/java/com/example/volunteer/opportunities_activity.java

```diff
@@ -0,0 +1,14 @@
+package com.example.volunteer;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class opportunities_activity extends AppCompatActivity {
```

Comment:
nit: naming convention for such files I believe would be better as:
`OpportunitiesActivity.java` and this class would be named `OpportunitiesActivity extends AppCompatActivity`

Reply:
<img width="860" alt="Screen Shot 2020-04-01 at 7 21 27 PM" src="https://user-images.githubusercontent.com/28017034/78204280-0a760800-744e-11ea-9f05-8b7c6078bb3c.png">